MekHQ/src/mekhq/campaign/mission/AtBDynamicScenarioFactory.java

```diff
@@ -2304,26 +2325,41 @@ public static void populateAeroBombs(List<Entity> entityList, Campaign campaign,
             if (entity.isBomber()) {
                 // if this entity has no guns (e.g. is a Boeing Jump Bomber)
                 if (entity.getIndividualWeaponList().size() == 0) {
-                    loadBombs(entity, validBombChoices);
+                    loadBombs(entity, validBombChoices, campaign.getGameYear());
                     continue;
                 }
 
                 if (numBombers >= maxBombers) {
                     break;
                 }
 
-                loadBombs(entity, validBombChoices);
+                loadBombs(entity, validBombChoices, campaign.getGameYear());
                 numBombers++;
             }
         }
     }
 
-    // todo: determine how many bombs can fit on this entity, and either pick
-    // a random amount or fill 'er up
-    private static void loadBombs(Entity entity, int[] validBombChoices) {
+    /**
+     * Worker function that takes an entity and an array of bomb types
+     * and loads it up with a random amount of bombs that it's capable of holding
+     */
+    private static void loadBombs(Entity entity, int[] validBombChoices, int year) {
         int[] bombChoices = new int[BombType.B_NUM];
+        
+        // remove bomb choices if they're not era-appropriate
+        List<Integer> actualBombChoices = new ArrayList<>();
+        for (int x = 0; x < validBombChoices.length; x++) {
+            String typeName = BombType.getBombInternalName(validBombChoices[x]);
+            
+            // hack: make rocket launcher pods available before 3055
+            if (validBombChoices[x] == BombType.B_RL ||
+                    BombType.get(typeName).isAvailableIn(year)) {
+                actualBombChoices.add(validBombChoices[x]);
+            }
+        }
+        
         int numBombs = (int) (entity.getWeight() / 5);
```

Comment:
This doesn't seem to randomize the number of bombs added?

Reply:
Updated with a bugfix, but looks like my intent was to give it a full bomb load, so comment updated.